package.json

```diff
@@ -26,7 +23,9 @@
   },
   "devDependencies": {
     "@briandipalma/eslint-config-bdp": "^3.0.0",
-    "babel": "^4.6.3",
+    "babel": "^6.5.2",
```

Comment:
Are we not meant to be using `babel-cli`, `babel` is deprecated in babel 6 https://github.com/babel/babel/blob/master/packages/babel/cli.js